docs/source/dataset_summary.rst

```diff
@@ -83,6 +83,17 @@ SSVEP
    :class:`Wang2016`,34,62,40,6,5s,250Hz,1
 
 
+c-VEP
+======================
+
+
```

Comment:
HI @thijor and @PierreGtch!

As much as the more traditional MOABB paradigms still don't have an introduction text, new datasets need to insert some general information, e.g. resting state dataset. For new and old users alike to understand that there is a change, they can use it for purpose.

Can you please add a sentence or two about the new dataset class? If you can also add some images, that would be great too. I think one figure simplified based on your article here would be great, like one of these two: 

![cvep](https://cfn-live-content-bucket-iop-org.s3.amazonaws.com/journals/1741-2552/18/5/056007/revision3/jneabeceff1_hr.jpg?AWSAccessKeyId=AKIAYDKQL6LTV7YY2HIK&Expires=1693081654&Signature=G5HMikS1GbaWrEkjbYwaYHaFIyU%3D)  

Please feel free to include a citation, if possible, which will help new users.

Reply:
I have added a sentence or two explaining the paradigm and a reference to a review article. Adding the specific image that you show is too much detail for the Thielen2021 dataset, not the c-VEP paradigm. I don't think you meant to add an explanation of the dataset, right?

Reply:
I was thinking about the paradigm but I am happy with the sentence. We should improve the doc and add images for each paradigm in another moment.